fre/yamltools/tests/test_combine_yamls.py

```diff
@@ -81,8 +81,8 @@ def test_combined_compileyaml_validation():
     Validate the combined compile yaml
     """
     combined_yamlfile =f"{COMP_OUT_DIR}/combined-{COMP_EXPERIMENT}.yaml"
-    schema_file = os.path.join(f"{IN_DIR}","compile_yamls","schema.json")
-
+    schema_file = os.path.join(Path(TEST_DIR).resolve().parents[1], "gfdl_msd_schemas", "FRE", "fre_make.json")
```

Comment:
similar comment RE `.parents` and, while we may move `gfdl_msd_schemas` idk who knows, might be clearer to just have the spot hard coded. will probably break more clearly than a `os.path` function call